lib/HTTP/Status.pm

```diff
@@ -91,6 +91,13 @@ my %StatusCode = (
     511 => 'Network Authentication Required', # RFC 6585: Additional Codes
 );
 
+require HTTP::Status::Unofficial;
```

Comment:
I'm not convinced that this is a good idea. It's easier to just leave the codes where they are in the main list.

Reply:
The idea I was toying with, was to make separate groups of codes in split files and that could be dynamically loaded.

Any new (proposed) RFC could than have it's own file, with it's own small documentation etc.

But ... lets leave that idea for now... me being ambitious